src/marketing/product-recommendations.md

```diff
@@ -21,7 +21,31 @@ This dashboard displays a table of previously configured recommendations (if any
 
 See [Recommendation Types]({% link marketing/prex-types.md %}) to learn about the available recommendation types in Magento.
 
-## Backup recommendations {#backup-recommendations}
+## Training the machine learning models
```

Comment:
Maybe "Time to train..." or "Training period for the" ? 
I want to communicate we talk about the "how long to wait?" question